.changeset/afraid-worms-perform.md

```diff
@@ -0,0 +1,7 @@
+---
+"wrangler": patch
+---
+
+feat: pass env value as `CLOUDFLARE_WRANGLER_ENV` to custom builds
```

Comment:
tbh, I'm not a fan of the name `CLOUDFLARE_WRANGLER_ENV`. Any suggestions on a better name?